### PR TITLE
Update every typescript to 5.2.0

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Review by augustjk — APPROVED** [review]
🎉

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
Please update `vue-tsc` to at least `1.8.8` (from `0.29.8`). That fixes `npm run test` from within `packages/labs/gen-wrapper-vue`.

**Comment by justinfagnani** [comment]
@AndrewJakubowicz I did update vue-tsc to 1.8.8 and still got the error

File: packages/labs/gen-wrapper-vue/src/lib/package-json-template.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const packageJsonTemplate = (
         '@vitejs/plugin-vue': '^3.1.2',
```
Comment (AndrewJakubowicz) [inline packages/labs/gen-wrapper-vue/src/lib/package-json-template.ts:42]: Non blocking: The generator sets TypeScript to `~4.7.4` as default. Should this also be bumped to TS 5.2?

**Review by AndrewJakubowicz — APPROVED** [review]
Nice!

File: packages/labs/gen-wrapper-vue/src/lib/package-json-template.ts (hunk 1 above)
Comment (justinfagnani) [reply packages/labs/gen-wrapper-vue/src/lib/package-json-template.ts:42]: Yeah, I'll bump that back up. We'll only have test coverage of one version at a time, so it should match what we test.

**Pull request merged**